libraries/Microsoft.Bot.Builder/IPropertyAccessor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder
+{
+
+    /// <summary>
+    /// This is metadata about the property including policy info
+    /// </summary>
+    public interface IPropertyAccessor
```

Comment:
I find the name "Property" to be far too abstract to stand on its own. At bare minimum, how about IStatePropertyAccessor?